src/core/services/gesture/gesture.js

```diff
@@ -79,8 +79,6 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
   var self = {
     handler: addHandler,
     register: register,
-    isIos: isIos,
```

Comment:
I think deleting it wasn't a nice thing. people may be using it outside of angular material. as a normal service which is capable of identifying if visitor is on iOS or Android.